gfbuild.sh

```diff
@@ -71,6 +71,9 @@ dev_build(){
 }
 
 build_re_dev(){
+  if [ -f "${WORKSPACE}/snapshots/pom.xml" ]; then
```

Comment:
This is probably because jakarta.el-api-5.0.0-RC1.jar has not yet been published on Maven Central.

https://repo.eclipse.org/content/repositories/maven_central/jakarta/el/jakarta.el-api/5.0.0-RC1/

Reply:
It tried to use even 5.0.0, not 5.0.0-RC1, probably the snapshot changed in between?
` Could not find artifact jakarta.el:jakarta.el-api:jar:5.0.0`

Reply:
Hmm, seems the Eclipse mirror is broken somehow ... and also the dependency on 5.0.0 instead on 5.0.0-RC1 is probably incorrect ... https://repo1.maven.org/maven2/jakarta/el/jakarta.el-api/5.0.0-RC1/

Reply:
Hmm, seems the Eclipse mirror is broken somehow ... and also the dependency on 5.0.0 instead on 5.0.0-RC1 is probably incorrect ... https://repo1.maven.org/maven2/jakarta/el/jakarta.el-api/5.0.0-RC1/

Reply:
... and the staging repo was removed from the hibernate-validator ...
https://jakarta.oss.sonatype.org/content/repositories/staging/jakarta/el/jakarta.el-api/5.0.0/